docs/integrations/webhooks/create-a-webhook.mdx

```diff
@@ -0,0 +1,261 @@
+---
+title: Create a Clerk webhook in your Next.js application
+description: Learn how to create a Clerk webhook with Svix.
+---
+
+# Create a Clerk webhook in your Next.js application
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk + Next.js application",
+      link: "https://clerk.com/docs/quickstarts/nextjs"
+    }
+  ]}
+>
+- Listen for an event in the Clerk Dashboard
+- Create a webhook handler that
+  - uses Svix to verify the webhook signature
```

Comment:
The two indented bullet point look really strange in the preview. A check mark and bullet.

Reply:
I brought it up to Brad in the #proj-docs-redesign channel: https://clerkinc.slack.com/archives/C06CA0JUZ8D/p1714489946533379